backend/fms_core/services/sample.py

```diff
@@ -353,12 +354,15 @@ def prepare_library(process: Process,
                 quality_flag=None
             )
 
-            new_derived_sample_data = {
-                "library_id": library.id
-            }
+
             derived_samples_destination = []
             volume_ratios = {}
+            # For pools of samples (a library for each derived sample)
```

Comment:
This is actually the generic way of doing it ;) should have been that way from the get-go. Let's blame the old self.